server/models/CoffeeShop.js

```diff
@@ -7,7 +7,7 @@ const CoffeeShopSchema = new Schema({
     required: true
   },
   founded: {
-    type: Date,
+    type: Number,
```

Comment:
I'm a little wary about storing a date as a number because it could possibly cause issues when trying to interact with that object in the front end when we're normally expecting strings, but this is fine if its the best way to do it.

Reply:
I'm ok with making it a string. I changed it to a Number b/c it had to agree with the GraphQLInt type. But that makes sense regarding the frontend, so I'll make it a string.

Reply:
If all we do is just displaying the year then I think string is fine

Reply:
Yep agreed! I fixed this issue in the push I just made.